makefiles/tools/targets.inc.mk

```diff
@@ -24,3 +24,8 @@ $(RIOTTOOLS)/mosquitto_rsmb/mosquitto_rsmb:
 
 mosquitto_rsmb: $(RIOTTOOLS)/mosquitto_rsmb/mosquitto_rsmb
 	@make -C $(RIOTTOOLS)/mosquitto_rsmb run
+
+$(RIOTTOOLS)/setsid/setsid: $(RIOTTOOLS)/setsid/Makefile
+	@echo "[INFO] setsid binary not found - building it from source now"
+	@make -C $(RIOTTOOLS)/setsid
+	@echo "[INFO] setsid binary successfully build!"
```

Comment:
Note to myself: s/build/built/

Reply:
Just checked and in fact, all other tools have the same typo. Let's leave it this way here and I'll open a follow-up PR to fix them all in a row.

Reply:
yep